src/ui/screens/ScreenSubscriptionInProgressWeb.qml

```diff
@@ -10,6 +10,17 @@ MZLoader {
     objectName: "subscriptionInProgressWeb"
     headlineText: MZI18n.PurchaseWebInProgress2
 
+    // Hack: The Component.OnCompleted event was not working for me (Bea).
+    // So I am running this on property initialization.
```

Comment:
@lesleyjanenorton @MattLichtenstein Do you know why the Component.onCompleted would not trigger here?

Reply:
missing an `import QtQuick 2.15` at the top 🙂